index.html

```diff
@@ -4237,6 +4237,38 @@ <h2 id="examples-tzigla">Tzigla</h2>
 
     <h2 id="changelog">Change Log</h2>
 
+    <b class="header">1.2.1</b> &mdash; <small><i>Jun. 3, 2015</i></small>
+    &mdash; <a href="https://github.com/jashkenas/backbone/compare/1.2.0...1.2.1">Diff</a>
+    &mdash; <a href="https://cdn.rawgit.com/jashkenas/backbone/1.2.1/index.html">Docs</a>
+    <br />
+    <ul style="margin-top: 5px;">
+      <li>
+        <tt>Collection#add</tt> will now avoid trying to parse a model instance with <tt>parse : true</tt>.
+      </li>
+      <li>
+        Bug fix where <tt>Collection#remove</tt> was not actually returning only the models
```

Comment:
ah funny.. I originally has misspelled "accurately" but it was changed to "actually"   The specific wording was meant to reflect the changelog line in `1.2.0` https://github.com/jashkenas/backbone/blob/master/index.html#L4317

But if it makes since to leave it vague, that's cool.  just " Bug fix in `Collection#remove` ?